packages/eslint-plugin-lit-a11y/template-analyzer/template-analyzer.js

```diff
@@ -65,16 +66,25 @@ class TemplateAnalyzer {
      */
     this.expressionValues = new Map();
 
-    /** @type {treeAdapter.DocumentType} */
-    // TODO: This is not ideal
-    // @ts-expect-error: parse5.DocumentFragment is a union with {}, so let's fudge this type for convenience
-    this._ast = parse5.parseFragment(this.templateExpressionToHtml(node), {
+    const html = this.templateExpressionToHtml(node);
+    const options = {
       treeAdapter,
       sourceCodeLocationInfo: true,
       onParseError: err => {
         this.errors.push(err);
       },
-    });
+    };
+
+    // parseFragment strips out html elements
+    // we do this to run checks on <html>
+    if (isRootElement(html)) {
+      this._ast = parse5.parse(html, options);
```

Comment:
I'd still like to make sure this doesn't mess up the error reporting further along in the templates. See my comment here: https://github.com/open-wc/open-wc/issues/1925#issuecomment-915834465

Reply:
Something else to consider here is that by changing the template analyzer, we're diverging and moving further away from the template analyzer upstream in https://www.npmjs.com/package/eslint-plugin-lit

We've recently had some discussions to see what the deltas between our template analyzer, and the original template analyzer are, and see if we can just import the original template analyzer instead. If we make changes here, it'll be harder to move back to the original template analyzer.

Maybe its worth making an issue on https://github.com/43081j/eslint-plugin-lit/issues to discuss supporting `<html>` tags in templates, and decide on a shared solution, so we don't diverge too far from the original template analyzer.